### ISSUE TITLE
Unable to sync

### ISSUE THREAD

**Issue by zeenqo** [semuxproject/semux#879]
Hi there,

Please note that this is an issue tracker reserved for bug reports and feature requests.

For general questions please use the Discord channel here: https://discord.gg/qQVckKZ

#### System information

Semux version: 1.1.1

OS & Version: `e.g., Windows 7 x64

#### Expected behaviour

Wallet should sync.

#### Actual behaviour

Wallet doesn't sync. Tried removing the database folder and sync from block #0 but still failed. Also tried downloading latest wallet version again, restarting my computer, restarting my router, but nothing helps.
Here is the debug log:

```
2018-05-20T18:59:08+0300 INFO     PubSub           PubSub service started
2018-05-20T18:59:12+0300 INFO     Kernel           Semux/v1.1.1-a089548/Windows/amd64
2018-05-20T18:59:12+0300 INFO     Kernel           System booting up: network = mainnet, networkVersion = 0, coinbase = ec58f585743246cc7f44a5c3ee12596088bc39bd
2018-05-20T18:59:12+0300 INFO     Kernel           Computer: manufacturer = LENOVO, model = 4334                            
2018-05-20T18:59:12+0300 INFO     Kernel           OS: name = Microsoft Windows 7 SP1 build 7601
2018-05-20T18:59:13+0300 INFO     Kernel           CPU: processor = Intel(R) Core(TM) i3-2310M CPU @ 2.10GHz, cores = 2 / 4
2018-05-20T18:59:13+0300 INFO     Kernel           Memory: total = 10183 MB, available = 7249 MB, swap total = 10182 MB, swap available = 10122 MB
2018-05-20T18:59:13+0300 INFO     Kernel           Disk: name = \\.\PHYSICALDRIVE0, size = 476937 MB
2018-05-20T18:59:13+0300 INFO     Kernel           Network: name = Qualcomm Atheros AR8152/8158 PCI-E Fast Ethernet Controller (NDIS 6.20), ip = []
2018-05-20T18:59:13+0300 INFO     Kernel           Network: name = Qualcomm Atheros AR9285 Wireless Network Adapter, ip = [192.168.1.11]
2018-05-20T18:59:13+0300 INFO     Kernel           Network: name = Teredo Tunneling Pseudo-Interface, ip = []
2018-05-20T18:59:13+0300 INFO     Kernel           Network: name = Microsoft ISATAP Adapter, ip = []
2018-05-20T18:59:13+0300 INFO     Kernel           Network: name = Microsoft ISATAP Adapter #3, ip = []
2018-05-20T18:59:13+0300 INFO     Kernel           Java: version = 1.8.0_161, xmx = 5248 MB
2018-05-20T18:59:14+0300 INFO     Kernel           Latest block number = 0
2018-05-20T18:59:14+0300 INFO     PeerClient       Use IP address: 127.0.0.1
2018-05-20T18:59:14+0300 INFO     PeerClient       Starting IP refresh thread
2018-05-20T18:59:14+0300 INFO     NodeManager      Node manager started
2018-05-20T18:59:14+0300 INFO     PeerServer       Starting peer server: address = 0.0.0.0:5161
2018-05-20T18:59:14+0300 INFO     PeerClient       New IP address detected: 127.0.0.1 => 84.111.61.158
2018-05-20T18:59:15+0300 INFO     SemuxBft         Consensus started
2018-05-20T18:59:15+0300 INFO     SemuxBft         Entered new_height: height = 1, # validators = 4
2018-05-20T18:59:16+0300 INFO     Kernel           Found a gateway device: local address = 192.168.1.11, external address = 84.111.61.158
```

It reaches this point and gets stuck.

**Comment by zeenqo** [comment]
@cryptokat just double checked and port is open.
@orogvany, firewall is turned off but I still can't sync.
It doesn't look like my computer blocks incoming messages on port 5161, because when I try to run an old wallet version I can get incoming messages, although they are invalid:

`
2018-05-20T19:29:23+0300 INFO     Kernel           Semux/v1.0.0-4a83a2d/Windows/amd64
2018-05-20T19:29:23+0300 INFO     Kernel           System booting up: network = mainnet, networkVersion = 0, coinbase = a2d3ba339c7f5c929f58a6d0827093ac640217a6
2018-05-20T19:29:24+0300 INFO     Kernel           Computer: manufacturer = LENOVO, model = 4334                            
2018-05-20T19:29:24+0300 INFO     Kernel           OS: name = Microsoft Windows 7 SP1 build 7601
2018-05-20T19:29:24+0300 INFO     Kernel           CPU: processor = Intel(R) Core(TM) i3-2310M CPU @ 2.10GHz, cores = 2 / 4
2018-05-20T19:29:24+0300 INFO     Kernel           Memory: total = 10183 MB, available = 6846 MB, swap total = 10182 MB, swap available = 10119 MB
2018-05-20T19:29:24+0300 INFO     Kernel           Disk: name = \\.\PHYSICALDRIVE0, size = 476937 MB
2018-05-20T19:29:25+0300 INFO     Kernel           Network: name = Qualcomm Atheros AR8152/8158 PCI-E Fast Ethernet Controller (NDIS 6.20), ip = []
2018-05-20T19:29:25+0300 INFO     Kernel           Network: name = Qualcomm Atheros AR9285 Wireless Network Adapter, ip = [192.168.1.11]
2018-05-20T19:29:25+0300 INFO     Kernel           Network: name = Teredo Tunneling Pseudo-Interface, ip = []
2018-05-20T19:29:25+0300 INFO     Kernel           Network: name = Microsoft ISATAP Adapter, ip = []
2018-05-20T19:29:25+0300 INFO     Kernel           Network: name = Microsoft ISATAP Adapter #3, ip = []
2018-05-20T19:29:25+0300 INFO     Kernel           Java: version = 1.8.0_161, xmx = 4958 MB
2018-05-20T19:29:26+0300 INFO     Kernel           Latest block number = 0
2018-05-20T19:29:26+0300 INFO     PeerClient       Use IP address: 127.0.0.1
2018-05-20T19:29:26+0300 INFO     PeerClient       Starting IP refresh thread
2018-05-20T19:29:27+0300 INFO     NodeManager      Node manager started
2018-05-20T19:29:27+0300 INFO     PeerServer       Starting peer server: address = 0.0.0.0:5161
2018-05-20T19:29:27+0300 INFO     SemuxBft         Consensus started
2018-05-20T19:29:27+0300 INFO     SemuxBft         Entered new_height: height = 1, # validators = 4
2018-05-20T19:29:28+0300 INFO     PeerClient       New IP address detected: 127.0.0.1 => 84.111.61.158
2018-05-20T19:29:28+0300 INFO     Kernel           Found a gateway device: local address = 192.168.1.11, external address = 84.111.61.158
2018-05-20T19:29:33+0300 INFO     SemuxP2pHandler  Received DISCONNECT message: reason = INVALID_HANDSHAKE, remoteIP = 167.99.16.150
2018-05-20T19:29:33+0300 INFO     SemuxP2pHandler  Received DISCONNECT message: reason = INVALID_HANDSHAKE, remoteIP = 159.203.147.157
2018-05-20T19:29:34+0300 INFO     SemuxP2pHandler  Received DISCONNECT message: reason = INVALID_HANDSHAKE, remoteIP = 139.162.48.46
2018-05-20T19:29:34+0300 INFO     SemuxP2pHandler  Received DISCONNECT message: reason = INVALID_HANDSHAKE, remoteIP = 172.104.98.36`

**Comment by cryptokat** [comment]
@zeenqo It seems that you are using a wireless connection. Have you tried to open the wallet with a wired connection?

**Comment by zeenqo** [comment]
I managed to solve the problem. My clock was lagging by 3 hours. When I fixed the clock, I was able to sync again. Maybe it's a good idea to add more informative messages to the debug log so it will warn that there is a problem with the date/time settings?

**Comment by cryptokat** [comment]
@zeenqo Thanks for the update. Closed in favour of #426

**Comment by orogvany** [comment]
Wireless should be fine, but weird no further logging. I wonder if we need some more logging or timeout logging to show whats going on

Changing base logger to debug in log4j file may yield additional insight
